tools/topology/topology2/include/pipelines/cavs/dai-copier-gain-module-copier-capture.conf

```diff
@@ -0,0 +1,112 @@
+#
+# BE capture pipeline: dai-copier-gain-module-copier-capture
+#
+# All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.dai-copier-gain-module-copier-capture.N.attribute_name"
+#
+# Usage: dai-copier-gain-module-copier-capture pipeline object can be instantiated as:
+#
+# Object.Pipeline.dai-copier-gain-module-copier-capture."N" {
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/gain.conf>
+<include/components/pipeline.conf>
+
+Class.Pipeline."dai-copier-gain-module-copier-capture" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		#
+		# dai-copier-gain-module-copier-capture objects instantiated
+		# within the same alsaconf node must have unique pipeline_id
+		# attribute
+		#
+		unique	"instance"
+	}
+
+	Object.Widget {
+		copier."1" {
+			type dai_out
+			period_sink_count 2
+			period_source_count 0
+			node_type $HDA_LINK_INPUT_CLASS
```

Comment:
why HDA_LINK_INPUT_CLASS ?

Reply:
> why HDA_LINK_INPUT_CLASS ?

This is copied from dai-copier-gain-mixin-capture.conf. It is a default value and will be overwrite in the .conf file where dai-copier-gain-module-copier-capture pipeline is used.

Reply:
you can delete it to remove any confusing question.

Reply:
> you can delete it to remove any confusing question.

ack, and removed